custom-recipes/sharepoint-online-append-list/recipe.py

```diff
@@ -0,0 +1,62 @@
+import dataiku
+from dataiku.customrecipe import get_input_names_for_role, get_recipe_config, get_output_names_for_role
+from safe_logger import SafeLogger
+from dss_constants import DSSConstants
+from sharepoint_client import SharePointClient
+
+
+logger = SafeLogger("sharepoint-online plugin", DSSConstants.SECRET_PARAMETERS_KEYS)
+logger.info('SharePoint Online append to list recipe v{}'.format(DSSConstants.PLUGIN_VERSION))
+
+
+def convert_date_format(json_row):
+    #  Convert pandas timestamps to iso
+    return json_row
```

Comment:
Nothing seems to be happening here

Reply:
Fixed in [13dc852](https://github.com/dataiku/dss-plugin-sharepoint-online/pull/47/commits/13dc852d214b9625a88b222fd20ebe6255dde56e)